docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: "3.7"
+version: "3.2"
```

Comment:
Not to nitpick, but does this actually use any `3.2` features?

Reply:
I think you're right @mcdonnnj.  I was thinking that secrets was added in 3.2, but it was actually added in 3.1.  Do you concur that the version should be 3.1?

Reply:
See commit 75388d5.

Reply:
`3.1` is what I was thinking as well.